### ISSUE TITLE
[UPnP Menu] Create Rated, Most played, Recently played?

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#2595]
Prerequisites: #2349

## Overvew

This is currently pending. 

![image](https://github.com/tesshucom/jpsonic/assets/27724847/782443e9-5a33-4db2-a4f5-9a36df9e755b)

## Details

Starred, Rated, Most played, Recently played are newly created, but the specifications are somewhat special.

#2593

#### Currently, these features are not very practical.

Rated, Most played, Recently played itself contains many unimplemented items. Therefore, we will need to solve them all. The problem is that, although the difficulty level is not high, these are not necessarily important functions in the overall system. Many engineers would rather spend the same amount of time doing something else. We don't know if an issue that has been ignored for decades will suddenly be treated as a high priority tomorrow.

This is a feature that users may wish to have. But, "Most Played", "Recently Played" fields exist in the table but are not used. They probably migrated to Scrobble because there was too much unnecessary processing. So handling it again could be like catching a falling knife.

Rated had been implemented halfway, but the web page does not have a UI, so if you want to use it, you will need to prepare the registration process first. Usually, when adding some functionality, Create, Read, Update, and Delete will be designed at the same time. That it is not done.

I'll put this on hold for now, but let me state that I understand that it's not a very high priority. And it is not certain whether it will be implemented or not. Of course, more important issues will always be dealt with first.
